### PR TITLE
Migrate comctl32 APIs

### PR DESCRIPTION
**Pull request by skdltmxn**

### CODE REVIEW

File: src/um/commoncontrols.rs [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        punkOuter: *const IUnknown,
+        riid: REFIID,
+        ppv: *mut *mut c_void,
+    ) -> HRESULT;
+}
+pub const ILIF_ALPHA: DWORD = 0x00000001;
+pub const ILIF_LOWQUALITY: DWORD = 0x00000001;
+pub const ILDRF_IMAGELOWQUALITY: LRESULT = 0x00000001;
+pub const ILDRF_OVERLAYLOWQUALITY: LRESULT = 0x00000010;
+RIDL!{#[uuid(0x46EB5926, 0x582E, 0x4017, 0x9F, 0xDF, 0xE8, 0x99, 0x8D, 0xAA, 0x09, 0x50)]
```
Comment (retep998) [inline src/um/commoncontrols.rs]: The uuid attribute should be lowercase hex: https://github.com/retep998/winapi-rs/blob/dev/CONTRIBUTING.md#com-interfaces

File: src/um/dpa_dsa.rs [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+        hdsa: HDSA,
+        i: c_int
+    ) -> BOOL;
+    pub fn DSA_DeleteAllItems(
+        hdsa: HDSA
+    ) -> BOOL;
+    pub fn DSA_EnumCallback(
+        hdsa: HDSA,
+        pfnCB: PFNDAENUMCALLBACK,
+        pData: *mut c_void
```
Comment (retep998) [inline src/um/dpa_dsa.rs]: Your functions are missing trailing commas.

**Comment by skdltmxn** [comment]
Fixed :)

**Comment by retep998** [comment]
Merge conflict.

**Comment by retep998** [comment]
And it's conflicted again because I just merged your other PR...

**Comment by skdltmxn** [comment]
And here's another merge!

**Pull request merged**